submissions/kolawoleOgunfowokan/HTML/js/script.js

```diff
@@ -0,0 +1,91 @@
+
+  let productArray = 
+  [
+      { 
+        image: "../images/HP250.jpg",
+        title: "HP Laptop",
+        price: "N130000"
+      },
+  
+      { 
+         image: "../images/eutetic-freezer.jpg",
+         title: "Freezer",
+         price: "N195000"
+       },
+  
+       {
+         image: "../images/huaweiMateBookXPro.jpg",
+         title: "Huwaei Laptop",
+         price: "N200000"
+       },
+  
+       {
+         image: "../images/java-prog.png",
+         title: "Java Book",
+         price: "N12000"
+       },
+  
+       {
+         image: "../images/microsoftSurface.jpg",
+         title: "Microsoft Surface",
+         price: "N320000"
+       },
+  
+       {
+         image: "../images/introprog.jpg",
+         title: "Introduction to Programming",
+         price: "N10000"
+       },
+  
+       {
+         image: "../images/Macbook.jpg",
+         title: "Macbook pro",
+         price: "N490000"
+       },
+  
+       {
+         image: "../images/AppleDesktop.png",
+         title: "Apple Desktop",
+         price: "N350000"
+       },
+  ];
+  
+  let button = document.getElementById("btn");
+  let displayContent = false;
```

Comment:
I think you can completely do without this flag. Maybe by just disabling the button after loading the products

Reply:
I tried, but it wouldn't work without it.

Reply:
Just disabling it was what I sweated about when I couldn't get it done. I had to set it to false globally, "let displayContent = false;"
Then set it to true even before the if statement in my function. The if statement then run if it is true.
However, after it has finished displaying once I clicked, I do not need the same product to display again on clicking the button, that is why I had to set,
products = button.disabled
which was more like saying
document.getElementById('product-box') = document.getElementById('btn').disabled
I just followed the logic the way I thought it out.
if I leave any of them out, the code breaks
@kenny-wealth

Reply:
`button.disabled` is just a boolean check that returns true or false depending on whether your button is disabled or not. You should try this in the console. So it doesn't disable the button. When the button is disabled, you should not be able to click it again. I think it you are compelled to use the `displayContent` flag because the button does not get disabled, hence the reason why your code breaks

Reply:
I think, I now get what you are talking about. The button shouldn't click at all. I have that now. You push hard @kenny-wealth.

Reply:
@kenny-wealth the code is neater now. Thanks for the push.